main/mafia.hs

```diff
@@ -281,7 +282,15 @@ pGhciEntryPoint =
     <> help "The entry point for GHCi."
 
 pGhciIncludes :: Parser [GhciInclude]
-pGhciIncludes = many (pGhciIncludeDirectory <|> pGhciIncludeAllLibraries)
+pGhciIncludes =
+  many (pGhciIncludeDirectory <|> pGhciIncludeProjectLibraries <|> pGhciIncludeAllLibraries)
```

Comment:
hmm that won't work as there is no `Monad` instance, not sure if something similar is possible though